collector/binlog.go

```diff
@@ -71,6 +78,11 @@ func ScrapeBinlogSize(db *sql.DB, ch chan<- prometheus.Metric) error {
 	ch <- prometheus.MustNewConstMetric(
 		binlogFilesDesc, prometheus.GaugeValue, float64(count),
 	)
+	// The last row contains the last binlog file number.
+	value, _ := strconv.ParseFloat(strings.Split(filename, ".")[1], 64)
+	ch <- prometheus.MustNewConstMetric(
+		binlogFileNumberDesc, prometheus.GaugeValue, value,
```

Comment:
It's a Gauge, as you care about the absolute value, it doesn't reset to 0 on process start and it can go down by operator action